Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+{
+    "id": "6B519857-1A20-4248-BAE8-AAD039015AFB",
+    "name": "AutomatedAnalysis",
+    "friendlyName": "Automated Analysis",
+    "description": "Do a automated analysis of your appication metrics ",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-monitor",
```

Comment:
Fix helpUrl, helpMarkDown and releaseNotes.

Reply:
I will fix this. Task documentation is not ready yet

Reply:
Getting task documentation merged will take time. For now we can specify the URL as the URL of the task folder. I'll add a README file on the task folder as well containing the documentation

Reply:
Pointing help to https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/kubernetes/canary-demo?view=azure-devops until documentation is ready.